src/content/docs/apm/apm-ui-pages/monitoring/kubernetes-summary-page.mdx

```diff
@@ -171,8 +171,6 @@ The following attributes are required in the APM service entity to display the K
 * `k8s.namespace.name`
 
 <Callout variant="caution">
-The Kubernetes summary page won't work properly if there is a mix of different instrumentation providers (New Relic and OpenTelemetry). For it to work properly, both Kubernetes and APM need to be monitored either exclusively through New Relic's proprietary agents or entirely through OpenTelemetry. 
-
-**Support for different instrumentation providers setup is expected by 2025 Q1.**
+The Kubernetes summary page works with a mix of instrumentation providers (New Relic and OpenTelemetry). However, if Kubernetes uses OpenTelemetry and APM uses New Relic, install `nri-metadata-injection` for it to work properly by providing the necessary Kubernetes metadata to the APM metrics.
```

Comment:
Can we change the callout type to info?

Can we include a link to the metadata injection Helm chart?
https://github.com/newrelic/k8s-metadata-injection/tree/main/charts/nri-metadata-injection